### DISCUSSION TITLE
Translations

### DISCUSSION THREAD

**Discussion by Zavy86** [Zavy86/WikiDocs#90] [Contributors]
Hi, this post is for all translators.
I will insert a message below every time the keys to be translated are changed.

**Comment by Zavy86** [comment]
Commit: 2b88d4290e292b0f0cff501d19830de980b3d90f

Translated:
- [x] `EN` - English (default)
- [x] `CS` - Česky
- [x] `DE` - Deutsch
- [ ] `ES` - Español
- [x] `FA` - فارسی
- [x] `FR` - Français
- [x] `JA` - 日本語
- [x] `IT` - Italian
- [ ] `NL` - Nederlands
- [x] `PL` - Polski
- [ ] `PT` - Português
- [ ] `RU` - Русский
- [x] `ZH-HANS` 中文(简体)
- [x] `ZH_HANT` 中文(繁體)

**Reply by ics01** [reply]
If you want help for french, I may help.

**Reply by Zavy86** [reply]
thank you, yes you can update the FR file and submit a pull request!
you can check in the tagged commit for new key added to be translated..

**Reply by ics01** [reply]
With pleasure ! As I am not familiar to Github, I only fond fr.json as a french file to translate. As some lines were missing, I finished to translate them. Then I created a new fork and a new pull request. As it is the first time, not sure it's the right way ?

**Reply by Zavy86** [reply]
yes! it's the right way! thank you so much!!

**Reply by ics01** [reply]
Cool, it was super-easy ! As I suscribed to this page, I will be informed on any change, and could continue translation.

**Reply by amaduain** [reply]
I can review spanish, let me give it a look.

**Reply by Zavy86** [reply]
awesome!

**Comment by Zavy86** [comment]
you only have to translate this file

https://github.com/Zavy86/WikiDocs/blob/master/localizations%2Fde.json

**Reply by Jesse-Jens** [reply]
I found the JSON files which need translating. Though Dutch seems to be translated already, I'd suggest slight changes but it seems good. Let me know if it needs work still.

**Reply by Zavy86** [reply]
when we add some keywords we add it in english into all files, for now it's ok, when we updati it you will see the language unflagged in this post! thank you
